### ISSUE TITLE
Would love to help with the Go guide

### ISSUE THREAD

**Issue by anubhakushwaha** [coderplex/learn#35]
Currently the course for Go just has titles, I would love to work upon adding code snippets for each and help in the content for each of the titles. Let me know if I can work upon this.

**Comment by anubhakushwaha** [comment]
@duttakapil Can I go ahead with this?

**Comment by duttakapil** [comment]
Yes, of course @anubhakushwaha go ahead. If you can recommend learning material for each title, along with providing a task or challenge to practice, that would be very helpful. Towards the end of the guide, we should also suggest some open source projects that use Go Lang and provide some additional resources for learning. Really appreciate you coming forward to contribute :D

**Comment by duttakapil** [comment]
Also, have you joined [our chatroom](https://chat.coderplex.org) yet @anubhakushwaha?

**Comment by elevenpassin** [comment]
@anubhakushwaha Hey, are you willing to still work on this issue?
